pkg/util/specifieddelete/specified_delete.go

```diff
@@ -18,29 +18,43 @@ package specifieddelete
 
 import (
 	"context"
-	"fmt"
 
-	appsv1alpha1 "github.com/openkruise/kruise/apis/apps/v1alpha1"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	appsv1alpha1 "github.com/openkruise/kruise/apis/apps/v1alpha1"
+	"github.com/openkruise/kruise/pkg/util"
 )
 
 func IsSpecifiedDelete(obj metav1.Object) bool {
 	_, ok := obj.GetLabels()[appsv1alpha1.SpecifiedDeleteKey]
 	return ok
 }
 
-func PatchPodSpecifiedDelete(c client.Client, pod *v1.Pod, value string) (bool, error) {
+func ShouldKeepPVC(obj metav1.Object) bool {
+	return obj.GetLabels()[appsv1alpha1.KeepPVCForDeletionKey] == "true"
```

Comment:
is it necessary to introduce an extra label ? can we just checking the following logic?
!cs.Spec.ScaleStrategy.DisablePVCReuse && utilfeature.DefaultFeatureGate.Enabled(features.CloneSetPVCReuseDuringUpdate)